dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/Contentlet.java

```diff
@@ -277,20 +249,72 @@ public void setLanguageId(long languageId) {
         this.languageId = languageId;
     }
 
+    /**
+     * @deprecated Use the method {@link #getContentTypeId()}
+     *
+     * @return
+     */
     public String getStructureInode() {
-        return structureInode;
+        return contentTypeId;
+    }
+
+    /**
+     * Returns the ID of the Content Type that this Contentlet belongs to.
+     *
+     * @return The Content Type ID.
+     */
+    public String getContentTypeId() {
+        return this.contentTypeId;
     }
 
+    /**
+     * @deprecated Use the method {@link #setContentTypeId(String)}
+     *
+     * @param structureInode
+     */
     public void setStructureInode(String structureInode) {
-        this.structureInode = structureInode;
+        this.contentTypeId = structureInode;
     }
+
+    /**
+     * Sets the Content Type ID that this Contentlet belongs to.
+     *
+     * @param contentTypeId The Content Type ID.
+     */
+    public void setContentTypeId(final String contentTypeId) {
+        this.contentTypeId = contentTypeId;
+    }
+
     /**
+     * @deprecated Use the method {@link #getContentType()}
+     *
+     * @return
      */
     public Structure getStructure() {
-        Structure structure = CacheLocator.getContentTypeCache().getStructureByInode(structureInode);
+        Structure structure = CacheLocator.getContentTypeCache().getStructureByInode(contentTypeId);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'structure' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5120047)